### DISCUSSION TITLE
Welcome to DepthAverageCurrents_SOP Discussions!

### DISCUSSION THREAD

**Discussion by soerenthomsen** [OceanGlidersCommunity/DepthAverageCurrents_SOP#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LewisDrysdale** [comment]
Hello, I am working with Seaglider data from the [UK OSNAP](https://www.ukosnap.org/observations) program. We have a near-continuous glider occupation of the Rockall Trough west of Scotland for around 3 years. I am presently trying to think about ways to improve position estimates when the glider is yoyo diving (which we do a lot to lessen then risk of fishing boats trawling the glider). The UoW seaglider software does not seem to estimate the yoyo dive surface position well because only information on north/east displacement is available, so we are using interpolated GPS fixes to avoid a 'crooked' track. I would be particularly interested in if anyone has calculated DACs using yoyo dives before. Shall I start a new thread?

**Comment by soerenthomsen** [comment]
Hi @LewisDrysdale 

thanks a lot for your interest. Indeed at GEOMAR we also did these yoyo dives to avoid collisions with fishing boats in upwelling regimes. 

I suggest you open an issue and after discussing this with the lead authors (@AlicePietri, @AnthonyBosse and @rtodd-WHOI and the community we should aim to implement also something related to yoyo dives /DAC calculation into the SOP together? 

Best 
Soeren

**Reply by LewisDrysdale** [reply]
Hi @soerenthomsen,

Sounds good. 

Cheers,

Lewis
